changelogs/DP-22656.yml

```diff
@@ -37,5 +37,5 @@
 #    issue: DP-19843
 #
 Changed:
-  - description: Migrated Our Organizations data to sections for Organization pages.
```

Comment:
@arthurbaghdas I think this changelog is suppose to be for DP-22658 instead of DP-22656. It overwrote another changelog file. Can you fix it?

Reply:
Thanks for noticing that. Fixed now